docs/source/configuration.rst

```diff
@@ -53,7 +57,8 @@ To specify the time-axis binning alone, do:
 
   report = df.pm_stability_report(time_axis='date', time_width='1w', time_offset='2020-1-6')
 
-The default time width is 4 weeks ('4w'). All other features (except for 'date') are auto-binned in this example.
+The default time width is 30 days ('30d'), with time offset 2014-1-4 (a Monday).
```

Comment:
Time offset is: `"bin_offset": pd.Timestamp("2010-01-04").value` according to the code.

Reply:
If provide a string like '2020-1-1' it gets automatically converted to that.

Reply:
I meant the date itself, it is: `2010-01-04` in the code and `2014-01-04` in your documentation.